client/src/contexts/userContext.js

```diff
@@ -113,6 +117,84 @@ const UserProvider = (props) => {
         });
       }
     },
+    addConversation: async (members) => {
+      try {
+        const res = axios.post('/conversation/add', members);
+        if (res.status !== 200)
+          return dispatch({
+            type: USER_ERROR,
+            payload: 'Error creating conversation',
+          });
+
+        const data = res.data;
+        console.log('data:', data);
+        dispatch({ type: ADD_CONVERSATION, payload: data });
+        return data;
+      } catch (err) {
+        dispatch({
+          type: USER_ERROR,
+          payload: 'Error creating conversation',
+        });
+      }
+    },
+    fetchSingleConversation: async (members) => {
+      const membersArr = members.map((member) => member._id);
+      const memberIDs = membersArr.join('&');
+      console.log('memberIDs:', memberIDs);
+      try {
+        const res = await axios.get(`/conversation/room/${memberIDs}`);
```

Comment:
instead of something like this: `/conversation/room/123&234`

`/conversation/room?members=[123,234]`